common/lib/xmodule/xmodule/assetstore/assetmgr.py

```diff
@@ -51,12 +44,4 @@ def find(asset_key, throw_on_not_found=True, as_stream=False):
         """
         Finds a course asset either in the assetstore -or- in the deprecated contentstore.
```

Comment:
This docstring is no longer accurate: we're not looking for the course asset in the `assetstore` anymore; we're only looking for it in the contentstore.